testsuite/MDAnalysisTests/coordinates/test_xyz.py

```diff
@@ -114,13 +150,53 @@ def __init__(self):
 
 
 class Test_XYZBZReader(TestXYZReader):
-    def __init__(self):
-        super(Test_XYZBZReader, self).__init__(XYZ_BZ_Reference())
+
+    @staticmethod
+    @pytest.fixture()
+    def ref():
+        return XYZ_BZ_Reference()
+
+    @staticmethod
+    @pytest.fixture()
+    def reader(ref):
+        reader = ref.reader(ref.trajectory)
+        reader.add_auxiliary('lowf', ref.aux_lowf, dt=ref.aux_lowf_dt, initial_time=0, time_selector=None)
+        reader.add_auxiliary('highf', ref.aux_highf, dt=ref.aux_highf_dt, initial_time=0, time_selector=None)
+        return reader
 
 
 class Test_XYZBZWriter(TestXYZWriter):
-    def __init__(self):
-        super(Test_XYZBZWriter, self).__init__(XYZ_BZ_Reference())
+
+    @staticmethod
+    @pytest.fixture()
+    def ref():
+        return XYZ_BZ_Reference()
+
+    @staticmethod
+    @pytest.fixture()
+    def reader(ref):
+        return ref.reader(ref.trajectory)
+
+    @staticmethod
+    @pytest.fixture()
+    def u_no_resnames():
```

Comment:
These Universe fixtures don't need to be defined in each format's tests, they're format agnostic.  I would either, define them in the base class (`BaseWriterTest`?) or just call them in the test function.  They're very light, so we're unlikely to have them has a large-scoped fixture.